src/workerd/api/tests/js-rpc-flag.js

```diff
@@ -12,13 +12,21 @@ export class DurableObjectExample {
   foo() {
     return 123;
   }
+
+  fetch(req) {
+    return new Response(req.method + " " + req.url);
```

Comment:
```suggestion
    return new Response(`${req.method} ${req.url}`);
```

Reply:
Agreed that's an improvement. Don't think it's worth restarting CI, though.